UPGRADE-3.0.md

```diff
@@ -595,6 +595,24 @@ UPGRADE FROM 2.x to 3.0
       }
    }
    ```
+   
+   If you're using `PUT` requests you need to pass the request method to the form:
+
+   Before:
+
+   ```php
+   $form = $this->createForm(FormType::class, $entity);
+   $form->submit($request);
+   ```
+
+   After:
+
+   ```php
+   $form = $this->createForm(FormType::class, $entity, [
+       'method' => $request->getMethod(),
```

Comment:
wouldn't it be better to put a static `"PUT"` value here, instead of letting the request configure it?

Reply:
When utilizing this for POST and PUT actions the current code would be more fitting, but perhaps it's more clear when using a static PUT value. Not sure what's better in this case. What do you think?

Reply:
The current code means allowing any method. Not sure this is what ppl should do. Anyone else?

Reply:
Imho, you should explicitly set the method, the form is meant to be for. Otherwise the form would trigger handling, validation etc. for the wrong method, while this very check is the first being made and early returns if not match. By setting the current method, you execute way too much unnecessary code.

Reply:
@fnagel up to add PUT explicitly here to match the first sentence in your comment ?